packages/testing/src/index.ts

```diff
@@ -129,8 +237,26 @@ export class TestWorkflowEnvironment {
     // TODO: the server should return exit code 0
     await kill(this.serverProc, 'SIGINT', { validReturnCodes: [0, 130] });
   }
+
+  /**
+   * Wait for `durationMs` in "test server time".
+   *
+   * The test server toggles between skipped time and normal time dependnding on what it needs to execute.
+   *
+   * This method is likely to resolve in less than `durationMs` of "real time".
+   *
+   * Useful for simulating events far into the future like completion of long running activities.
+   *
```

Comment:
A motivating example would be helpful here I think.

Reply:
👍